addons/binding/org.openhab.binding.rfxcom/ESH-INF/thing/channels.xml

```diff
@@ -170,4 +170,24 @@
 		<state min="0" max="360" step="1" readOnly="true"></state>
 	</channel-type>
 
+	<channel-type id="scene">
+		<item-type>Number</item-type>
+		<label>Scene</label>
+		<description>Scene channel</description>
+		<state readOnly="false">
+			<options>
+				<option value="0">Scene 1</option>
```

Comment:
Regarding the rules, just tested these:

```
Switch SceneSwitch
Number LivoloScene

rule "SceneOn"
when
	Item SceneSwitch received command ON
then
	LivoloScene.send(new DecimalType(0))
end

rule "SceneOff"
when
	Item SceneSwitch received command OFF
then
	LivoloScene.send(new DecimalType(1))
end
```

Scene channel's item type is Number, so we have to pass a DecimalType. I've intentionally designed it this way, as use of string values in interfaces is usually considered to be a bad practice.

Although rules allow to mimic a scene switch, I insist to keep it on Java side for the reasons mentioned above.